app/models/option_select_facet.rb

```diff
@@ -42,10 +42,6 @@ def has_filters?
     selected_values.any?
   end
 
-  def close_facet?
```

Comment:
I understand your point about not having this decision be made in the model. It's something that would be better suited in a presenter rather than a view. Also, presenter tests are just unit tests, and therefore much easier to write!